src/components/AqiGauge/AqiDial.tsx

```diff
@@ -80,20 +80,22 @@ const AqiDial: ({currentAqi}: DialProps) => JSX.Element = ({
   const {t} = useTranslation(['dial', 'menu']);
 
   return (
-    <Box>
-      <Center>
-        <GaugeSvg currentAqi={currentAqi} />
-      </Center>
-      <Text fontSize={30}>{t('aqi') + currentAqi}</Text>
-      <Text fontSize={14} mb={2}>
-        {t('moreInfo')}
-        <Link fontSize={14} color="#32bfd1" href="/health">
-          {' '}
-          {t('menu:healthInfo')}
-        </Link>
-      </Text>
-      <AqiLabel currentAqi={currentAqi} />
-    </Box>
+    <Flex height="100%" width="100%" justifyContent="center" align="center">
```

Comment:
Just added this flex to center the dial

Reply:
If the goal is to center the dial, couldn't the containing Box just become the Flex instead?